HackYourFuture-CPH/Guide-IT
# chages file names on folders components and container acording to gui…

…delines

changed file names on folders components and container acording to guidelines: https://github.com/HackYourFuture-CPH/Guide-IT#coding-guidelines-for-components.
Also changes paths on those specific files if needed and in App.js
Made the neccessary testing to make sure everything works as before.


Fixes #93 

# How to test?

Check visually the file and folder names to make sure that are named according to the guidelines.
Run npm run storybook and check that all components work fine.
Run npm run dev and you should see app workin fine on localhost:8080
# Checklist
